fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/CanonicalValue.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.util;
+
+/**
+ * A canonical value, with processing applied to identify the uri, version, and fragment
+ */
+public class CanonicalValue {
```

Comment:
wondering if this one should get a equals and hashcode impl so it can be used in more places

Reply:
done